new-contract-interaction/agent-config.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I don't think we need this file since we have the `agent-config.json.example` file